perl.c

```diff
@@ -4226,6 +4226,18 @@ S_open_script(pTHX_ const char *scriptname, bool dosearch, bool *suidscript)
                 scriptname = savepv(s + 1);
                 Safefree(PL_origfilename);
                 PL_origfilename = (char *)scriptname;
+            } else {
```

Comment:
Our current style favors un-cuddling the `else`, *i.e.,* `else` should begin a new line.